<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_menu</class>
 <widget class="QWidget" name="gestion_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>761</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter </string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>641</width>
       <height>371</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifient</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Composition</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_composition">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>100</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="le_date">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher </string>
    </attribute>
    <widget class="QTableView" name="tab_menu1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>561</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_pdf_menu">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>GÃ©neration:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Modifier </string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Identifient</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Composition</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string> Date</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_id_modif">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nom_modif">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="la_composition_modif">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>200</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>452</x>
       <y>167</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="le_date_modif">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>260</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Supprimer </string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Identifient</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_id_supp">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>91</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>170</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
